ssg/build_remediations.py

```diff
@@ -146,6 +151,81 @@ def get_populate_replacement(remediation_type, text):
     sys.exit(1)
 
 
+def from_file(file_path, env_yaml):
+    mod_file = []
+    config = defaultdict(lambda: None)
+
+    fix_file_lines = jinja_process_file(file_path, env_yaml).splitlines()
+
+    # Assignment automatically escapes shell characters for XML
+    for line in fix_file_lines:
+        line += "\n"
+        if line.startswith('#'):
+            try:
+                (key, value) = line.strip('#').split('=')
+                if key.strip() in ['complexity', 'disruption',
+                                   'platform', 'reboot',
+                                   'strategy']:
+                    config[key.strip()] = value.strip()
+                else:
+                    if not line.startswith(FILE_GENERATED):
+                        mod_file.append(line)
+            except ValueError:
+                if not line.startswith(FILE_GENERATED):
+                    mod_file.append(line)
+        else:
+            mod_file.append(line)
+
+    return mod_file, config
+
+
+def process_fix(fixes, args, env_yaml, product, file_path, fix_name):
```

Comment:
Scrutinizer is right about arguments, but it is difficult to do something sensible with it without heavy refactoring. At least passing `args` is confusing from the caller perspective, as it is not obvious that the function takes a look only at remediation type.